designcompose/src/main/java/com/android/designcompose/LiveUpdateExceptions.kt

```diff
@@ -0,0 +1,3 @@
+package com.android.designcompose
+
+class AccessDeniedException: RuntimeException()
```

Comment:
What does this do?

Reply:
Just declares an exception class so we can catch and handle specific ones.